.pre-commit-config.yaml

```diff
@@ -5,6 +5,12 @@ repos:
       - id: pyupgrade
         args: ["--py37-plus"]
 
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v2.0.0
+    hooks:
+      - id: autoflake
+        args: ["--in-place", "--imports"]
```

Comment:
Why are you using the `--imports` flag?
The `--in-place` is to make changes in the code right?